arch/arm/boot/dts/nxp/imx/Makefile

```diff
@@ -411,10 +420,5 @@ dtb-$(CONFIG_SOC_IMX7D) += \
 dtb-$(CONFIG_SOC_IMX7ULP) += \
 	imx7ulp-com.dtb \
 	imx7ulp-evk.dtb
-dtbo-$(CONFIG_SOC_IMX6UL) += imx6ul-ts7250v3-silo104-i2c-gpio.dtbo
 dtb-$(CONFIG_SOC_IMXRT) += \
 	imxrt1050-evk.dtb
-
-# Allow overlays to be sourced from .dtso files too
-$(obj)/%.dtbo: $(src)/%.dtso FORCE
-	$(call if_changed,dtc_overlay)
```

Comment:
It looks like this is reverting changes added in 8d49c841657eeb58f9640495e6a5529ae184eb91. If the 8d49c841657eeb58f9640495e6a5529ae184eb91 commit was not necessary, these should be squashed so only the singular needed change is made as opposed to multiple commits back to back adding and then removing code.